Sources/PodToBUILD/BuildFile.swift

```diff
@@ -156,16 +81,24 @@ public struct AcknowledgmentNode: SkylarkConvertible {
     public func toSkylark() -> SkylarkNode {
         let nodeName = bazelLabel(fromString: name + "_acknowledgement").toSkylark()
         let options = GetBuildOptions()
+        let value: String
         let podSupportBuildableDir = String(PodSupportBuidableDir.utf8.dropLast())!
-        let value = (getRulePrefix(name: options.podName) +
-             podSupportBuildableDir +
-             ":acknowledgement_fragment").toSkylark()
+        if options.path == "." {
+            value = (getRulePrefix(name: options.podName) +
+                        podSupportBuildableDir + ":acknowledgement_fragment")
+        } else {
+            // TODO: This will not work with external. Consider moving this file to
+            // pod_support instead.
+            value = "//\(options.path)/\(podSupportBuildableDir):acknowledgement_fragment"
```

Comment:
Do we need a issue filed for this?

Reply:
Yep, we'll need another ticket to make this work with the Bazel workspace rule, which would require this. I'd like to get rid of the directory anyhow but the PR is getting out of hand already